pykusto/functions.py

```diff
@@ -387,7 +387,8 @@ def pack_dictionary(): raise NotImplemented  # TODO
 # def parse_ipv4(self): return
 
 
-def parse_json(): raise NotImplemented  # TODO
+def parse_json(expr: Union[StringType, DynamicType]) -> MappingExpression:
```

Comment:
The return type doesn't have to be a mapping, it could also be an array, so most generally a `Union[MappingExpression, ArrayExpression]`

Reply:
I would also add test for that